frontend/src/views/auth/LoginView/LoginFrom.css

```diff
@@ -0,0 +1,11 @@
+form {
```

Comment:
Pamietaj tylko o przejsciu na CSS modules, ale to juz wspominalem na discordzie.